### PR TITLE
Add Actueel scaffolding

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

Nothing much yet, just these preparations for the actueel pages:

- Add Actueel skeleton pages for national, region and municipal sections
- Make 'actueel' the default page name so that all choropleths by default will link to to the 'actueel' page.
- Actueel has become the national homepage, latest developments has moved to a separate route (laatste-ontwikkelingen) 

## Motivation

Feature request

## Detailed design

Summary pretty much says it all

## Drawbacks

n/a

## Alternatives

n/a

## Unresolved questions

n/a

### Governance

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,98 +1,36 @@
...
-  escalationLevel1: number;
-  escalationLevel2: number;
-  escalationLevel3: number;
-  escalationLevel4: number;
-  escalationLevel5: number;
-};
-
-const Home: FCWithLayout<INationalHomepageData> = (props) => {
-  const { data, text, escalationLevelCounts } = props;
+const Home: FCWithLayout<IHomeData> = (data) => {
```
Comment (VWSCoronaDashboard8) [inline src/pages/index.tsx:26]:
```suggestion
const HomePage: FCWithLayout<HomePageProps> = (data) => {
```

Comment (VWSCoronaDashboard8) [reply src/pages/index.tsx:26]: I know we're currently not using the Page suffix on (most) pages, but I think it makes things a bit more explicit, and it avoids this I prefix thing.

File: src/pages/landelijk/laatste-ontwikkelingen.tsx [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+ */
+type EscalationLevelCounts = {
+  escalationLevel1: number;
+  escalationLevel2: number;
+  escalationLevel3: number;
+  escalationLevel4: number;
+  escalationLevel5: number;
+};
+
+const LatestDevelopments: FCWithLayout<ILatestDevelopmentsData> = (props) => {
```
Comment (VWSCoronaDashboard8) [inline src/pages/landelijk/laatste-ontwikkelingen.tsx:55]:
```suggestion
const LatestDevelopmentsPage: FCWithLayout<LatestDevelopmentsPageProps> = (props) => {
```

File: src/pages/veiligheidsregio/[code]/actueel.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  getSafetyRegionData,
+  getSafetyRegionPaths,
+  ISafetyRegionData,
+} from '~/static-props/safetyregion-data';
+import { parseMarkdownInLocale } from '~/utils/parse-markdown-in-locale';
+import { EscalationMapLegenda } from '..';
+
+type ActueelData = ISafetyRegionData & { text: TALLLanguages };
+
+const SafetyRegionActueel: FCWithLayout<ActueelData> = (data) => {
```
Comment (VWSCoronaDashboard8) [inline src/pages/veiligheidsregio/[code]/actueel.tsx:20]:
```suggestion
const SafetyRegionActueelPage: FCWithLayout<SafetyRegionActueelPageProps> = (data) => {
```

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: .gitignore [hunk 4]
```diff
@@ -13,6 +13,11 @@ node_modules
 
 # CI/CD
 /env
+/public/json
 .gitlab-ci.yml
 
 /*.code-workspace
+/cypress/screenshots
+/cypress/videos
```
Comment (VWSCoronaDashboard8) [inline .gitignore]: These files are not present in the mono repo root I think? Shouldn't these be added to the app .gitignore instead?

File: packages/app/src/components-styled/data-driven-text.tsx [hunk 5]
```diff
@@ -45,7 +40,10 @@ export function DataDrivenText({
       .join(':')}`
   );
 
-  const differenceValue = differenceKey
+  const differenceValue:
+    | DifferenceInteger
+    | DifferenceDecimal
+    | undefined = differenceKey
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/components-styled/data-driven-text.tsx:46]: Difference values are never undefined according to the data type definitions, so I wonder what this means and why it was necessary.

File: packages/app/src/components-styled/layout/app-footer.tsx [hunk 6]
```diff
@@ -59,6 +59,7 @@ export function AppFooter({ lastGenerated }: { lastGenerated: string }) {
 }
 
 function LastGeneratedMessage({ date }: { date: string }) {
+  console.log(date);
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/components-styled/layout/app-footer.tsx]: console log

File: packages/app/src/components-styled/link-with-icon.tsx [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+          textDecoration: 'none',
+          '&:hover': { textDecoration: 'underline' },
+        })}
+      >
+        <span
+          css={css({
+            svg: {
+              height: '10px',
+              width: '12px',
+              transform: 'rotate(90deg)',
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/components-styled/link-with-icon.tsx:27]: this transform shouldn't be here of course .. instead it should rotate icons before passing them on these props.

**Pull request merged**